### PR TITLE
Updates on Top Women page

### PR DESCRIPTION
**Pull request by flaviogalizia**

### CODE REVIEW

File: sites/diverseeducation.com/server/templates/website-section/awards-honors/top-women/index.marko [hunk 1]
```diff
@@ -116,7 +116,7 @@ $ const queryParams = {
             flush=true
             full-height=true
             with-teaser=false
-            with-section=false
+            with-section=falseb
```
Comment (Shinsina) [inline sites/diverseeducation.com/server/templates/website-section/awards-honors/top-women/index.marko:119]: Is this a typo/stray character addition here?

**Review by Shinsina — CHANGES_REQUESTED** [review]
https://github.com/parameter1/cox-matthews-associates-websites/blob/master/sites/diverseeducation.com/server/graphql/fragments/section-with-children.js#L8

This will need to be changed from `{ field: sequence, order: asc }` to `{ field: name, order: desc }` and that should resolve the sorting issue. Do double check to see the other pages that use this file don't have their sorts suddenly broken (this appears to be used for Arthur Ashe, Emerging Scholars and Graduate Scholars as well) and that the build passes once corrected.

**Comment by Shinsina** [comment]
[PR #210](https://trello.com/c/DyC3TF7G/274-pr-210)

File: sites/diverseeducation.com/server/templates/website-section/awards-honors/top-women/index.marko (hunk 1 above)
Comment (flaviogalizia) [reply sites/diverseeducation.com/server/templates/website-section/awards-honors/top-women/index.marko:119]: I haven't edited that part of the code, anyway, it's been corrected.

Comment (flaviogalizia) [reply sites/diverseeducation.com/server/templates/website-section/awards-honors/top-women/index.marko:119]: I updated the section-with-children.js file, still doesn't look right.

File: sites/diverseeducation.com/server/templates/website-section/awards-honors/top-women/config/top-women-yearly/2023.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  title: 'Celebrating Leading Women In Higher Education',
+  subtitle: '<i>Diverse</i> presents 25 women making a difference in the world of academe.',
+  //createdBy: 'Compiled by Walter Hudson',
```
Comment (Shinsina) [inline sites/diverseeducation.com/server/templates/website-section/awards-honors/top-women/config/top-women-yearly/2023.js:4]: This needs to be `// createdBy: 'Compiled by Walter Hudson',` or preferably just removed

File: sites/diverseeducation.com/server/templates/website-section/awards-honors/top-women/index.marko (hunk 1 above)
Comment (Shinsina) [reply sites/diverseeducation.com/server/templates/website-section/awards-honors/top-women/index.marko:119]: This stray character is still here.

**Review by Shinsina — CHANGES_REQUESTED** [review]

**Review by Shinsina — APPROVED** [review]

**Pull request merged**